index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Erol Kucuk Microverse Portfolio">
+  <title>Microverse Erol Portfolio</title>
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="styles.css"> 
+
+</head>
+
+<body>
+  <!-- Header -->
+  <header class="header">
+    <div class="leftsection" href="#"><img src="images/My Logo.png"></a></div>
+     <nav class="rightsection">
+      <a class="iconmenu" href="#"><img src="images/Enabled.png"></a>
+     </nav>
+  </header>
+  <!-- End Header -->
+
+  <!-- Introduction Area -->
+  <main class="headline">
+    <h1 class="introduction"> i'm Erol Glad to see you!</h1>
+    <p class="intro-description">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+      works.
+      If you like what you see and have a project you need coded, do not hesitate and contact me.
+    </p>
+    <!-- End Introduction Area -->
+    <p class="support">LET'S CONNECT</p>
+    <ul class="social-media">
+      <li><a href="#"><img src="images/Vector (1).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector (2).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector (3).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector (4).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Linkedin icon.svg" alt=""></a></li>
+
+
+
+     </ul>
+     </main>
+</body>
```

Comment:
##### Good job making most of the changes @erolkck, Although there are some issues still to be addressed. 
- [ ] For the font, I see you apply `Impact`, but this is not the right font as the Figma design defines. Please use `Poppins`.
- [ ] For the background there are still some missing elements. please add those. This will improve your skill to stick to a provided design.


Template | Your design
-- | --
![figma](https://user-images.githubusercontent.com/69077061/205379777-ede05897-888f-4df4-b692-943652048daf.PNG)| ![you](https://user-images.githubusercontent.com/69077061/205379802-9c135ac7-85d2-4ca8-849a-27855c89aa4f.PNG)